nullaway/src/main/java/com/uber/nullaway/ErrorProneCLIFlagsConfig.java

```diff
@@ -288,6 +292,7 @@ final class ErrorProneCLIFlagsConfig implements Config {
         getPackagePattern(
             getFlagStringSet(flags, FL_EXCLUDED_FIELD_ANNOT, DEFAULT_EXCLUDED_FIELD_ANNOT));
     castToNonNullMethod = flags.get(FL_CTNN_METHOD).orElse(null);
+    legacyAnnotationLocation = flags.getBoolean(FL_LEGACY_ANNOTATION_LOCATION).orElse(false);
```

Comment:
I think this method is where you could do the sanity check to ensure that legacy annotation mode and JSpecify mode are not simultaneously enabled.

Reply:
Added